src/app/files/edit-location.component.ts

```diff
@@ -6,7 +6,7 @@ import { Location } from './location';
 @Component({
     selector: 'edit-location',
     template: `
-        <div>
+        <div class="sme-focus-zone">
```

Comment:
We don't need to use sme-focus-zone and arrow key can be used instead.

Reply:
tested, seem to still work